TestPamCLIRunStandalone fails locally

The test fails when an authd instance is running on the system executing the tests. Tried both on my host running Ubuntu 26.04 and in a Ubuntu 24.04 VM:

```
go test ./pam/integration-tests/... -run TestPamCLIRunStandalone -count 1
--- FAIL: TestPamCLIRunStandalone (1.97s)
    cli_test.go:457: (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.061: Starting authenticate
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.061: Called with arguments: '/tmp/pam-cli-tester-2282947005/exec-child' 'debug=true' 'connection_timeout=' '--exec-debug' 'logfile=/dev/stdout'
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.061: Setting up connection at unix:abstract=authd-authd-pam-runner-service-3b3fe8c600f3c6a50ae1e9166a0dd49f (3b3fe8c600f3c6a50ae1e9166a0dd49f)
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.061: Server started, connectable address unix:abstract=authd-authd-pam-runner-service-3b3fe8c600f3c6a50ae1e9166a0dd49f
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.061: Environment:
          AUTHD_PAM_SERVER_ADDRESS=unix:abstract=authd-authd-pam-runner-service-3b3fe8c600f3c6a50ae1e9166a0dd49f
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.061: Launching '/tmp/pam-cli-tester-2282947005/exec-child -flags 32768 authenticate debug=true connection_timeout= logfile=/dev/stdout'
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.062: Launched child 1430794
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.066: Accepting connection from PID 1430794 on connection 3b3fe8c600f3c6a50ae1e9166a0dd49f
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.066: authenticate: called method GetData(('authd.already-authenticated-flag',))
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.066: authenticate: called method GetItem((1,))
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.066: authenticate: called method GetItem((3,))
        17:34:55 WARN Impossible to parse connection timeout "", using default!
        17:34:55 DEBUG Connecting to unix:///run/authd.sock
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.067: authenticate: called method UnsetData(('authd.authentication-broker-id',))
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.068: authenticate: called method GetItem((1,))
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.068: authenticate: called method GetEnvList(())
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.068: authenticate: called method GetItem((1,))
        17:34:55 DEBUG adapter.initHealthCheck{}
        17:34:55 DEBUG adapter.supportedUILayoutsReceived{layouts:[]*authd.UILayout{(*authd.UILayout)(0xc00012a690), (*authd.UILayout)(0xc00012a700), (*authd.UILayout)(0xc00012a770)}}
        17:34:55 DEBUG Native model update: adapter.supportedUILayoutsReceived{layouts:[]*authd.UILayout{(*authd.UILayout)(0xc00012a690), (*authd.UILayout)(0xc00012a700), (*authd.UILayout)(0xc00012a770)}}
        17:34:55 DEBUG Native model update: adapter.supportedUILayoutsSet{}
        17:34:55 DEBUG adapter.pamError{status:4, msg:"could not get current available brokers: permission denied: only root can perform this operation"}
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.069: authenticate: called method Prompt((3, 'could not get current available brokers: permission denied: only root can perform this operation'))
        PAM Error Message: could not get current available brokers: permission denied: only root can perform this operation
        17:34:55 DEBUG LOGIN: exiting with error System error: could not get current available brokers: permission denied: only root can perform this operation
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.101: Waiting pid 1430794, returned 1430794, exited: 1, signaled: 0
        (pam-runner_authd-pam-runner-service-authenticate:1430114): authd-pam-exec-DEBUG: 17:34:55.101: Child 1430794 exited with exit status 4 (System error)
        PAM Authenticate()
          User: "user"
          Result: error: PAM exit code: 4
            System error
        (pam-runner_authd-pam-runner-service-acct_mgmt:1430114): authd-pam-exec-DEBUG: 17:34:55.101: Starting acct_mgmt
        (pam-runner_authd-pam-runner-service-acct_mgmt:1430114): authd-pam-exec-DEBUG: 17:34:55.102: Called with arguments: '/tmp/pam-cli-tester-2282947005/exec-child' 'debug=true' 'connection_timeout=' '--exec-debug' 'logfile=/dev/stdout'
        (pam-runner_authd-pam-runner-service-acct_mgmt:1430114): authd-pam-exec-DEBUG: 17:34:55.102: Environment:
          AUTHD_PAM_SERVER_ADDRESS=unix:abstract=authd-authd-pam-runner-service-3b3fe8c600f3c6a50ae1e9166a0dd49f
        (pam-runner_authd-pam-runner-service-acct_mgmt:1430114): authd-pam-exec-DEBUG: 17:34:55.102: Launching '/tmp/pam-cli-tester-2282947005/exec-child -flags 32768 acct_mgmt debug=true connection_timeout= logfile=/dev/stdout'
        (pam-runner_authd-pam-runner-service-acct_mgmt:1430114): authd-pam-exec-DEBUG: 17:34:55.103: Launched child 1430805
        (pam-runner_authd-pam-runner-service-acct_mgmt:1430114): authd-pam-exec-DEBUG: 17:34:55.107: Accepting connection from PID 1430805 on connection 3b3fe8c600f3c6a50ae1e9166a0dd49f
        (pam-runner_authd-pam-runner-service-acct_mgmt:1430114): authd-pam-exec-DEBUG: 17:34:55.107: acct_mgmt: called method GetItem((1,))
        (pam-runner_authd-pam-runner-service-acct_mgmt:1430114): authd-pam-exec-DEBUG: 17:34:55.107: acct_mgmt: called method GetData(('authd.authentication-broker-id',))
        17:34:55 DEBUG AcctMgmt: exiting with error The return value should be ignored by PAM dispatch
        (pam-runner_authd-pam-runner-service-acct_mgmt:1430114): authd-pam-exec-DEBUG: 17:34:55.107: Waiting pid 1430805, returned 1430805, exited: 1, signaled: 0
        (pam-runner_authd-pam-runner-service-acct_mgmt:1430114): authd-pam-exec-DEBUG: 17:34:55.107: Child 1430805 exited with exit status 25 (The return value should be ignored by PAM dispatch)
        PAM Info Message: acct=incomplete
        PAM AcctMgmt()
          User: "user"
          Result: error: PAM exit code: 25
            The return value should be ignored by PAM dispatch
        
    cli_test.go:460: Expected output to contain Authentication service cannot retrieve authentication info
FAIL
FAIL	github.com/canonical/authd/pam/integration-tests	2.912s
FAIL
```

It's passing in CI.

--- a/pam/integration-tests/testdata/golden/TestCLIAuthenticate/Deny_authentication_if_current_user_is_not_considered_as_root
+++ b/pam/integration-tests/testdata/golden/TestCLIAuthenticate/Deny_authentication_if_current_user_is_not_considered_as_root
@@ -2,8 +2,8 @@
 PAM Error Message: could not get current available brokers: permission denied: only root can perform this operation
 PAM Authenticate()
   User: "<unset>"
-  Result: error: PAM exit code: 4
-    System error
+  Result: error: PAM exit code: 9
+    Authentication service cannot retrieve authentication info
 PAM Info Message: acct=incomplete
 PAM AcctMgmt()
   User: "<unset>"
@@ -15,8 +15,8 @@ PAM AcctMgmt()
 PAM Error Message: could not get current available brokers: permission denied: only root can perform this operation
 PAM Authenticate()
   User: "<unset>"
-  Result: error: PAM exit code: 4
-    System error
+  Result: error: PAM exit code: 9
+    Authentication service cannot retrieve authentication info
 PAM Info Message: acct=incomplete
 PAM AcctMgmt()
   User: "<unset>"

--- a/pam/integration-tests/testdata/golden/TestCLIChangeAuthTok/Prevent_change_password_if_current_user_is_not_root_as_can_not_authenticate
+++ b/pam/integration-tests/testdata/golden/TestCLIChangeAuthTok/Prevent_change_password_if_current_user_is_not_root_as_can_not_authenticate
@@ -2,8 +2,8 @@
 PAM Error Message: could not get current available brokers: permission denied: only root can perform this operation
 PAM ChangeAuthTok()
   User: "<unset>"
-  Result: error: PAM exit code: 4
-    System error
+  Result: error: PAM exit code: 9
+    Authentication service cannot retrieve authentication info
 PAM Info Message: acct=incomplete
 PAM AcctMgmt()
   User: "<unset>"
@@ -15,8 +15,8 @@ PAM AcctMgmt()
 PAM Error Message: could not get current available brokers: permission denied: only root can perform this operation
 PAM ChangeAuthTok()
   User: "<unset>"
-  Result: error: PAM exit code: 4
-    System error
+  Result: error: PAM exit code: 9
+    Authentication service cannot retrieve authentication info
 PAM Info Message: acct=incomplete
 PAM AcctMgmt()
   User: "<unset>"

--- a/pam/integration-tests/testdata/golden/TestNativeAuthenticate/Deny_authentication_if_current_user_is_not_considered_as_root
+++ b/pam/integration-tests/testdata/golden/TestNativeAuthenticate/Deny_authentication_if_current_user_is_not_considered_as_root
@@ -2,8 +2,8 @@
 PAM Error Message: could not get current available brokers: permission denied: only root can perform this operation
 PAM Authenticate()
   User: "user-integration-native-deny-authentication-if-current-user-is-not-considered-as-root@example.com"
-  Result: error: PAM exit code: 4
-    System error
+  Result: error: PAM exit code: 9
+    Authentication service cannot retrieve authentication info
 acct=incomplete
 PAM AcctMgmt()
   User: "user-integration-native-deny-authentication-if-current-user-is-not-considered-as-root@example.com"
@@ -15,8 +15,8 @@ PAM AcctMgmt()
 PAM Error Message: could not get current available brokers: permission denied: only root can perform this operation
 PAM Authenticate()
   User: "user-integration-native-deny-authentication-if-current-user-is-not-considered-as-root@example.com"
-  Result: error: PAM exit code: 4
-    System error
+  Result: error: PAM exit code: 9
+    Authentication service cannot retrieve authentication info
 acct=incomplete
 PAM AcctMgmt()
   User: "user-integration-native-deny-authentication-if-current-user-is-not-considered-as-root@example.com"

--- a/pam/integration-tests/testdata/golden/TestNativeChangeAuthTok/Prevent_change_password_if_current_user_is_not_root_as_can_not_authenticate
+++ b/pam/integration-tests/testdata/golden/TestNativeChangeAuthTok/Prevent_change_password_if_current_user_is_not_root_as_can_not_authenticate
@@ -2,8 +2,8 @@
 PAM Error Message: could not get current available brokers: permission denied: only root can perform this operation
 PAM ChangeAuthTok()
   User: "<unset>"
-  Result: error: PAM exit code: 4
-    System error
+  Result: error: PAM exit code: 9
+    Authentication service cannot retrieve authentication info
 acct=incomplete
 PAM AcctMgmt()
   User: "<unset>"
@@ -15,8 +15,8 @@ PAM AcctMgmt()
 PAM Error Message: could not get current available brokers: permission denied: only root can perform this operation
 PAM ChangeAuthTok()
   User: "<unset>"
-  Result: error: PAM exit code: 4
-    System error
+  Result: error: PAM exit code: 9
+    Authentication service cannot retrieve authentication info
 acct=incomplete
 PAM AcctMgmt()
   User: "<unset>"

--- a/pam/internal/adapter/brokerselection.go
+++ b/pam/internal/adapter/brokerselection.go
@@ -160,7 +160,7 @@ func getAvailableBrokers(client authd.PAMClient) tea.Cmd {
 		brokersInfo, err := client.AvailableBrokers(context.TODO(), &authd.Empty{})
 		if err != nil {
 			return pamError{
-				status: pam.ErrSystem,
+				status: pam.ErrAuthinfoUnavail,
 				msg:    fmt.Sprintf("could not get current available brokers: %v", err),
 			}
 		}

--- a/pam/internal/adapter/gdmmodel_test.go
+++ b/pam/internal/adapter/gdmmodel_test.go
@@ -1704,7 +1704,7 @@ func TestGdmModel(t *testing.T) {
 				gdm.EventType_userSelected,
 			},
 			wantExitStatus: pamError{
-				status: pam.ErrSystem,
+				status: pam.ErrAuthinfoUnavail,
 				msg:    "could not get current available brokers: brokers loading failed",
 			},
 			wantNoBrokers: true,

